arex-instrumentation/redis/arex-redission-v316/src/main/java/io/arex/inst/redisson/v316/wrapper/CommandAsyncServiceAdviceWrapper.java

```diff
@@ -0,0 +1,162 @@
+package io.arex.inst.redisson.v316.wrapper;
+
+import io.arex.inst.redis.common.RedisKeyUtil;
+import io.arex.inst.redis.common.redisson.RedissonHelper;
+import io.arex.inst.redis.common.redisson.RedissonWrapperCommon;
+import java.util.Arrays;
+import java.util.List;
+import org.redisson.SlotCallback;
+import org.redisson.api.RFuture;
+import org.redisson.client.RedisClient;
+import org.redisson.client.codec.Codec;
+import org.redisson.client.protocol.RedisCommand;
+import org.redisson.command.CommandAsyncService;
+import org.redisson.connection.ConnectionManager;
+import org.redisson.connection.MasterSlaveEntry;
+import org.redisson.liveobject.core.RedissonObjectBuilder;
+import org.redisson.liveobject.core.RedissonObjectBuilder.ReferenceType;
+
+public class CommandAsyncServiceAdviceWrapper extends CommandAsyncService {
```

Comment:
```java
public class Redisson implements RedissonClient {
    protected final CommandAsyncExecutor commandExecutor;
    commandExecutor = new CommandSyncService(connectionManager, objectBuilder);
}
```
v315  public class CommandAsyncServiceAdviceWrapper extends `CommandSyncService {`

v316 public class CommandAsyncServiceAdviceWrapper extends `CommandAsyncService {`

Please make sure it's not a mistake.

Reply:
fixed